### DISCUSSION TITLE
raspberry pi4 ubuntu

### DISCUSSION THREAD

**Discussion by damasb** [mbround18/valheim-docker#636] [Help!] [answered]
hello, thank you very much for this project 
I'm new to this (docker I mean)

steps that I did 
but I think I have done everything right but can't connect to the server (I don't see even ports "listening")

**Steps that I ran to install:**
1 - installed steamcmd with https://github.com/tonistiigi/binfmt
2 -created user steam 1000:1000
3 - created docker-compose.yml
4 - I run docker-compose up --detach 
Logs
[Instalation](https://pastebin.com/xHcswx66)
1 - valheim_server.err it's empty
2 - valheim_server.log
it stays on the "Preloaded 'libsteam_api.so'" no more information shows up 
the only error shown is the "Error: unable to set group"

ODIN][INFO]  - Executing: /home/steam/valheim/valheim_server.x86_64 .....
[ODIN][INFO]  - --------------------------------------------------------------------------------------------------------------
[ODIN][INFO]  - Executable: /home/steam/valheim/valheim_server.x86_64
[ODIN][INFO]  - Launching Command...
[ODIN][INFO]  - Everything looks good! Running normally!
[ODIN][ERROR] - Error: unable to set group
Mono path[0] = '/home/steam/valheim/valheim_server_Data/Managed'
Mono config path = '/home/steam/valheim/valheim_server_Data/MonoBleedingEdge/etc'
Preloaded 'libparty.so'
Preloaded 'libsteam_api.so'
==> /home/steam/valheim/logs/valheim_server.err <==
==> /home/steam/valheim/logs/output.log <==
==> /home/steam/valheim/logs/auto-update.out <==
==> /home/steam/valheim/logs/auto-backup.out <==

and it stops here all the time

**Comment by mbround18** [comment, marked as answer]
@damasb I am unsure about this but it might be that the user you are running docker as doesnt have those permissions ? also the docker image has steamcmd in it

Pi4 is ARM this container & valheim will not run on ARM architecture
